simtools/schemas/model_parameters/store_photoelectrons.schema.yml

```diff
@@ -0,0 +1,41 @@
+%YAML 1.2
+---
+title: Schema for store_photoelectrons model parameter
+version: 0.1.0
+meta_schema: simpipe-schema
+meta_schema_url: https://raw.githubusercontent.com/gammasim/simtools/main/simtools/schemas/model_parameter_and_data_schema.metaschema.yml
+meta_schema_version: 0.1.0
+name: store_photoelectrons
+description: |-
+  While the save_photons=2 configuration option will save photo-electron data for all events,
+  this parameter allows to set a lower limit on the number of photo-electrons
+  which must get registered before deciding to store them.
+  Setting it to a value of zero or bigger (default: -1) will automatically enable the corresponding save_photons bit.
+  It should be cross-checked that the min_photons parameter is unlikely to result
+  in fewer than the given number of photo-electrons, as no attempt is made to trace photons through the telescope
+  and produce any photo-electrons if there are too few photons in the first place.
+data:
+  - type: int
+    unit: byte
```

Comment:
Is this unit correct?

Reply:
No, it's in p.e., but that's not a valid unit, so I set it to dimensionless.